docs/source/topics/concept.rst

```diff
@@ -0,0 +1,136 @@
+The Job Queue is filled with :term:`jobs <Job>` in creation-time order by the
+:term:`Job Generator`. The queue is managed by the :term:`scheduler` and contains
+all :term:`jobs <Job>` that are not yet scheduled to a :term:`drone` as well as
+:term:`jobs <Job>` that have not yet been processed succesfully.
+
+Pools
+~~~~~
+
+Pools are created based on the pool input files. Each pool is characterised by
+a set of defined resources. Further, pools have a ``capacity`` number of
+:term:`drones <Drone>` that can be created from a given pool. If the capacity
+is not specified, a maximum capacity of ``float("inf")`` is assumed.
+
+For pools, we differentiate static and dynamic pools. While static pools are
+intialised with a fixed amount of :term:`drones <Drone>`, the number of
+:term:`drones <Drone>` is adapted dynamically by the
+:term:`pool controller <Controller>` for dynamic pools.
+
+.. autoclass:: lapis.pool.Pool
+.. autoclass:: lapis.pool.StaticPool
+
+Controllers
+~~~~~~~~~~~
+
+Each :term:`pool` is started with an appropriate controller. Each controller runs
+periodically to check :term:`allocation` and :term:`utilisation` of assigned
+:term:`pool(s) <Pool>` to regulate the demand of :term:`drones <Drone>` for the
+given :term:`pool`.
+
+The concept of controllers is introduced by COBalD. The controllers implemented
+in LAPIS share the general concept as well as implementation by subclassing
+provided controllers such as :py:class:`cobald.controller.linear.LinearController`
+or :py:class:`cobald.controller.relative_supply.RelativeSupplyController` and
+overwriting :py:meth:`lapis.controller.SimulatedLinearController.run`. In
+this way, we enable validation of current TARDIS/COBalD setup as well as simulation
+of future extensions.
+
+Available controller implementations from COBalD in LAPIS are:
+
+.. autoclass:: lapis.controller.SimulatedLinearController
+    :members:
+
+.. autoclass:: lapis.controller.SimulatedRelativeSupplyController
+    :members:
+
+And there is also an implementation considered as an extension for COBalD:
+
+.. autoclass:: lapis.controller.SimulatedCostController
+    :members:
+
+Drones
+~~~~~~
+
+Drones provide instances of the set of resources defined by a given :term:`pool`.
+Drones are the only objects in the simulation that are able to process
+:term:`jobs <Job>`. Simplified, drones represent worker nodes.
+
+The concept of drones is introduced by TARDIS. A drone is a generalisation of
+the pilot concept used for example in High Energy Physics and is a placeholder
+for the real workloads to be processed. A drone is expected to autonomously
+manage its lifecycle, meaning, that it handles failures and termination
+independently from other components within the system.
+
+.. warning::
+
+    Drones are not yet fully employed in LAPIS. They already run independently
+    but do not handle termination themselves.
+
+Scheduler
+~~~~~~~~~
+
+The scheduler is the connecting component between the :term:`jobs <Job>` in the
```

Comment:
This section is very technically detailed. Perhaps a more general description matching other scheduler implementations would be more appropriate.

Reply:
What do you mean? Could you clarify a bit?
My intention is to follow the descriptions before. Those followed the actual implementation within LAPIS, so this is what I also did here for the scheduler.